machine/v1beta1/types_azureprovider.go

```diff
@@ -397,6 +419,28 @@ type OSDiskManagedDiskParameters struct {
 	// DiskEncryptionSet is the disk encryption set properties
 	// +optional
 	DiskEncryptionSet *DiskEncryptionSetParameters `json:"diskEncryptionSet,omitempty"`
+	// SecurityProfile specifies the security profile for the managed disk.
```

Comment:
Should use the json tag version of the name in godoc
```suggestion
	// securityProfile specifies the security profile for the managed disk.
```